app/src/main/java/com/locoquest/app/RetrofitClientInstance.kt

```diff
@@ -3,16 +3,16 @@ package com.locoquest.app
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
-class RetrofitClientInstance {
```

Comment:
-Removed redundancy in retrofit2
-Added baseurl here
-changed from class to object

Reply:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.